.github/workflows/nessie.yaml

```diff
@@ -375,6 +375,7 @@ jobs:
       TAG: ${{ needs.deploy-image.outputs.tag }}
       # Setting Account_ID as a secret as a way to avoid specifying it here
       REPO: ${{ secrets.AWS_ACCOUNT_ID }}.dkr.ecr.us-east-1.amazonaws.com
+      ACTIONS_VAR: "this_is_actions_var"
```

Comment:
If possible, inject this env var from the test code and not from here - otherwise when running locally, one has to make sure the var is set in order for the test to work.

Reply:
lakeFS is running separately from the test code. I can't control it from the test code, I can set it always in lakeFS but that seems like too much. WDYT?